sunpy/time/tests/test_timerange.py

```diff
@@ -292,3 +292,10 @@ def test_intersects():
     tr2 = sunpy.time.TimeRange('2020-01-01', '2020-01-04')
     assert tr1.intersects(tr2)
     assert tr2.intersects(tr1)
+
+def test_range():
```

Comment:
Can I get a few more tests, with larger ranges, what happens if you pass in a negative one or 0 even.